apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1022,6 +1028,27 @@ export class RushConfiguration {
     return this._repositoryUrl;
   }
 
+  /**
+   * The default git branch of the repository. This helps "rush change" find the right remote to compare against.
+   */
+  public get repositoryDefaultBranch(): string {
+    return this._repositoryDefaultBranch;
+  }
+
+  /**
+   * The default git remote of the repository. This helps "rush change" find the right remote to compare against.
+   */
+  public get repositoryDefaultRemote(): string {
+    return this._repositoryDefaultRemote;
+  }
+
+  /**
+   * The default fully-qualified git remote of the repository. This helps "rush change" find the right remote to compare against.
```

Comment:
```suggestion
   * The default fully-qualified git remote branch of the repository. This helps "rush change" find the right branch to compare against.
```

Reply:
(this is relied to next change too)
remote is the stored string key for remote repository URL + settings relied to it; I do not know whether it can be named better. [Git docs](https://git-scm.com/docs/git-remote) does not give it other name rather than "remote"
> Manage the set of repositories ("remotes") whose branches you track.

Reply:
This property is specifically referring to the remote branch, though. See docs here: https://git-scm.com/book/en/v2/Git-Branching-Remote-Branches

Git refers to the <remote>/<branchname> as either "remote branch" or "remote tracking branch."

Reply:
ah, got it. my bad, I thought this is about an another line